### ISSUE TITLE
Advance configuration

### ISSUE THREAD

**Issue by allamand** [codenrhoden/ansible-role-rexray#2]
Hello,

is it possible to make advance configuration through ansible-rexray playbook ?

i would like to activate preemption : 

> To enable this to happen, you must enable the pre-emption feature on the hosts. `rexray.volume.mount.preempt: true`. http://rexray.readthedocs.io/en/stable/user-guide/config/#volume-configuration

I also would like to activate rexray debug mode if possible

thanks

**Comment by codenrhoden** [comment]
Yes, those additional config fields need to be added to the module still.  It's on my list of things to do, for sure!

I will be taking a good look at this again soon, as the release of rex-ray 0.4 adds a lot of additional ways that rex-ray can be configured.
